config/cp.php

```diff
@@ -7,14 +7,16 @@
     | Control Panel
     |--------------------------------------------------------------------------
     |
-    | Whether the Control Panel should be enabled, and through what route.
+    | Whether the Control Panel should be enabled, and through what route and domain.
     |
     */
 
     'enabled' => env('CP_ENABLED', true),
 
     'route' => env('CP_ROUTE', 'cp'),
 
+    'domain' => env('CP_DOMAIN', null),
```

Comment:
What happens if no `CP_DOMAIN` is set in the user's `.env`. Does the Control Panel break?

If it does, it may be a good idea to fallback to the `APP_URL`?

Reply:
I do not believe it will break because the [`Illuminate\Routing\Route::domain` documentation](https://laravel.com/api/9.x/Illuminate/Routing/Route.html#method_domain) shows that it accepts a null value.  And in the [source code](https://github.com/laravel/framework/blob/9.x/src/Illuminate/Routing/Route.php#L747) attached from the documentation, a value of `null` falls back to the default domain.